src/Module/ModuleOffersList.php

```diff
@@ -230,7 +214,7 @@ protected function compile(): void
             $this->Template->pagination = $objPagination->generate("\n  ");
         }
 
-        $objArticles = OfferModel::findItems($this->config, ($this->limit ?: 0), ($this->offset ?: 0));
+        $objArticles = Offer::findItems($this->config, ($this->limit ?: 0), ($this->offset ?: 0));
```

Comment:
Can we also get rid of `Article` here ?

Reply:
Done in 9a85b3c75fdfc8a6a1ca879b597e96a0443792c0